playbooks/empty-dir-quota.yaml

```diff
@@ -0,0 +1,7 @@
+- hosts: schedulable_nodes
+  gather_facts: yes
+  become: yes
+  vars_files:
+    - vars/main.yaml
+  roles:
+      - ../roles/openshift-emptydir-quota
```

Comment:
will probably be better to add a symlink to ../roles in the playbooks directory rather than reference the role this way.